lib/swaggerValidator/trafficValidator.ts

```diff
@@ -306,44 +345,39 @@ export class TrafficValidator {
         }
         return 0;
       });
-      const sortedUnCoveredOperationsListGen = unCoveredOperationsListFormat.reduce(
-        (res: unCoveredOperationsFormat[], item) => {
-          item.operationIdList.sort(function (op1, op2) {
-            const opId1 = op1.operationId;
-            const opId2 = op2.operationId;
-            if (opId1 < opId2) {
-              return -1;
-            }
-            if (opId1 > opId2) {
-              return 1;
-            }
-            return 0;
-          });
-          res.push({
-            operationIdList: item.operationIdList,
-          });
-          return res;
-        },
-        []
-      );
 
-      this.operationCoverageResult.push({
-        spec: key,
-        apiVersion: getApiVersionFromFilePath(key),
-        coveredOperaions: coveredOperaions,
-        coverageRate: coverageRate,
-        unCoveredOperations: value.length - coveredOperaions,
-        totalOperations: value.length,
-        validationFailOperations: validationFailOperations,
-        unCoveredOperationsList: sortedUnCoveredOperationsList,
-        unCoveredOperationsListGen: sortedUnCoveredOperationsListGen,
-      });
+      for (let i = 0; i < unCoveredOperationsListFormat.length - 1; i++) {
```

Comment:
Can you add comment for this sorting by ascending order?

Reply:
added